rosbag2_py/test/common.py

```diff
@@ -0,0 +1,13 @@
+import rosbag2_py._rosbag2_py as rosbag2_py
```

Comment:
This file will need a copyright block on top.

Reply:
Fixed!